charts/external-dns/templates/deployment.yaml

```diff
@@ -107,8 +110,13 @@ spec:
           resources:
             {{- toYaml . | nindent 12 }}
           {{- end }}
-      {{- with .Values.extraVolumes }}
       volumes:
+      {{- if .Values.secrets }}
```

Comment:
Do you not need to mount this volume?

Reply:
I need to mount, but the mount path is not standardized. For example aws provider and azure provider requires different mount location. Make no sense to standardize it.